subsys/bluetooth/mesh/net.c

```diff
@@ -54,6 +55,13 @@ LOG_MODULE_REGISTER(bt_mesh_net);
 #define SRC(pdu)           (sys_get_be16(&(pdu)[5]))
 #define DST(pdu)           (sys_get_be16(&(pdu)[7]))
 
+/* Information needed for bridging the network PDUs */
+struct pdu_ctx {
```

Comment:
Should this also be inside a #if IS_ENABLED....?

Reply:
I guess type is not mandatory to wrap up in conditional compilation since there are no any instances of it. Compiler will not generate any warnings.

Reply:
Yup. It does not generate warnings.